var/spack/repos/builtin/packages/sortmerna/package.py

```diff
@@ -16,6 +16,11 @@ class Sortmerna(CMakePackage):
     version('2017-07-13', commit='8bde6fa113a5d99a23ae81b48eeea6760e966094')
 
     depends_on('zlib')
+    depends_on('sse2neon', when='target=aarch64:')
+
+    patch('for_aarch64.patch', when='target=aarch64:')
+
+    build_directory = 'spack-build'
```

Comment:
Instead of changing this, let's use `self.build_directory` below instead of `'spack-build'`.

Reply:
You can remove this line now.